src/index.ts

```diff
@@ -37,18 +39,22 @@ export interface Bundle {
 export interface ExploreOptions {
   /** Exclude "unmapped" bytes from the output */
   onlyMapped?: boolean;
-  /** Generate html */
-  html?: boolean;
-  /** Generate html ans save to specified file path  */
-  file?: string;
+  /** Output result as a string */
+  output?: {
+    format: OutputFormat;
+    /** Filename to save output to */
+    filename?: string;
+  };
   /** Disable removing prefix shared by all sources */
   noRoot?: boolean;
+  /** Replace "what" by "that" map */
   replaceMap?: ReplaceMap;
 }
 
 export interface ExploreResult {
   bundles: ExploreBundleResult[];
-  html?: string;
+  /** Result as a string - either JSON, TSV or HTML */
+  output?: string;
```

Comment:
When would this be missing?

Reply:
When `ExploreOptions.output` isn't set (e.g. when user wants to work with raw JSON (`bundles` property)). See `output.ts:formatOutput`